src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -358,8 +358,10 @@ protected string GeneratePropertyName(string fieldName)
                 baseName = fieldName;
             }
 
-            // Make uppercase the first letter
-            return char.ToUpper(baseName[0]).ToString() + baseName.Substring(1);
+            // Make the first character upper case using the "en-US" culture.  See discussion at
+            // https://github.com/dotnet/roslyn/issues/5524.
+            var firstCharacter = CompletionRules.EnUSCultureInfo.TextInfo.ToUpper(baseName[0]);
```

Comment:
@davkean was suggesting that we have something like: `CultureFallbacks`, and also encapsulate the fallback comparison from @rchande's change there, so that all of that policy is in one place.

He also mentioned that we should use `CultureInfo.GetCulture()`, instead of `new CultureInfo()`.